JenkinsfileRT_dev

```diff
@@ -103,11 +103,6 @@ bc1.build_cmds = [
     "pip freeze",
 ]
 bc1.build_cmds = PipInject(env.OVERRIDE_REQUIREMENTS) + bc1.build_cmds
-bc1.test_cmds = [
```

Comment:
Wait, I thought we had decided to leave the Mac build and test active in the `dev` file, which gets run once a week by the devdeps test.

Reply:
I realized that this should be copied from the `bc0` anyway, so having it here is redundant; I'll check that it runs in the job.